src/include/sof/audio/component.h

```diff
@@ -164,6 +164,13 @@
 #define tracev_comp(__e, ...)	tracev_event(TRACE_CLASS_COMP, __e, ##__VA_ARGS__)
 /** @}*/
 
+/* \brief Type of endpoint this component is connected to in a pipeline */
+enum comp_endpoint_type {
+	ENDPOINT_HOST,
```

Comment:
I think the convention should be COMP_ENDPOINT_HOST, COMP_ENDPOINT _*

Reply:
added prefixes